packages/module/src/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+// ============================================================================
+// Chatbot Footer
+// ============================================================================
+
+// Global footer component
+// Supports:
+// - Message bar
+// - Footnote
+// - Footnote popover
+
+import React from 'react';
+
+export interface FooterProps {
```

Comment:
This should extend `React.HTMLProps<HTMLDivElement>`

PF usually includes a `className?: string` prop to allow people to pass custom class names as well.

Reply:
Updated.